perses/analysis/load_simulations.py

```diff
@@ -2,55 +2,77 @@
 import matplotlib.pyplot as plt
 import sys
 from openeye import oechem, oegraphsim
-import logging
-
-
-_logger = logging.getLogger("analysis")
-
-
-class Molecule(object):
-    def __init__(self, i, string):
-        from perses.utils.openeye import smiles_to_oemol
-        self.line = string
-        details = string.split(';')
-        self.index = i
-        self.smiles, self.name, self.exp, self.experr, self.calc, self.calcerr = details[1:7]
-        self.mol = smiles_to_oemol(self.smiles)
-        self.exp = kcal_to_kt(float(self.exp))
-        self.experr = kcal_to_kt(float(self.experr))
-        self.calc = kcal_to_kt(float(self.calc))
-        self.calcerr = kcal_to_kt(float(self.calcerr))
-        self.mw = self.calculate_molecular_weight()
-        self.ha = self.heavy_atom_count()
-        self.simtype = None
 
-    def calculate_molecular_weight(self):
-        """ Calculates the molecular weight of an oemol
+import logging
+_logger = logging.getLogger()
+_logger.setLevel(logging.INFO)
+_logger = logging.getLogger("load_simulations")
 
-        Parameters
-        ----------
 
-        Returns
-        -------
-        float, molecular weight of molecule
+class Simulation(object):
+    """Object that holds the results of a free energy simulation.
+    # TODO make this much more flexible
+    Assumes that the output is in the form lig{A}to{B}/out-{phase}.nc
 
-        """
-        return oechem.OECalculateMolecularWeight(self.mol)
+    This automatically loads the data and performs the data. Results can be accessed through attributes.
 
-    def heavy_atom_count(self):
-        """ Counts the number of heavy atoms in an oemol
+    >>> # Load and analyze the simulation for `lig0to1/out.*`
+    >>> simulation_results = Simulation(0,1)
+    >>> # Report the binding free energy
+    >>> print(f'Relative binding free energy: {simulation_results.bindingdg}')
 
-        Parameters
-        ----------
 
+    Parameters
+    ----------
+    A : int
+        integer of first ligand
+    B : int
+        integer of second ligand
 
-        Returns
-        -------
-        int, number of heavy atoms in molecule
-        """
-        return oechem.OECount(self.mol, oechem.OEIsHeavy())
+    Attributes
+    ----------
+    ligA : int
+        integer of first ligand
+    ligB : int
+        integer of first ligand
+    directory : string
+        output directory location : lig{A}to{B}
+    vacdg : float
+        relative free energy in vacuum phase (kcal/mol)
+    vacddg : float
```

Comment:
@hannahbrucemacdonald : These docstrings are now incorrect. According to the code, you are storing variances for  `vacddg`, `solddg`, and `comdd`, but standard errors for `hydrationddg` and `complexddg`.

What if you always stored standard errors instead, and remembered that you have to propagate the error in `A+B` as `sqrt(dA**2 + dB**2)`?